docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,226 @@
+
+You can fine-tune the frequency of the model persistence and the snapshot 
+retention days by setting the following configuration options:
+
+* `background_persist_interval` sets the time between each periodic persistence 
+  of the model. The default value is a randomized value between 3 to 4 hours, 
+  which avoids all jobs persisting at the same time. Increase this value 
+  appropriately to fit your job.
+
+* `daily_model_snapshot_retention_after_days` affects the automatic removal of 
+  old model snapshots for the job. It specifies a period of time (in days) after 
+  which only the first snapshot per day is retained. This period is relative to 
+  the timestamp of the most recent snapshot for this job. Consider setting this 
+  value to only a couple of days after which one snapshot per day will be 
+  retained.
+
+* `model_snapshot_retention_days` defines the time in days that model snapshots 
+  are retained for the job. The default value is ten days meaning snapshots ten 
+  days older than the newest snapshot are deleted.
+
+For more information, refer to <<ml-model-snapshots>>.
+
+TIP: When your job is shut down, and it is not closed properly, it rolls back to 
+the last successfully saved snapshot when it is started again. Close the job 
+gracefully when it is possible instead of killing it so that you can preserve 
+the exact model state.
+
+
+[[results-retention]]
+== Setting result retention time
+
+{anomaly-detect-cap} results are retained indefinitely by default. For big jobs, 
+results build up over time, and your result index will be quite large. Consider 
+how long you wish to retain the results and set `results_retention_days` 
+accordingly – for example, to 30 or 60 days – to avoid unnecessarily large 
+result indices.   
+
+
+[[renormalization]]
+== Configuring renormalization window days
+
+When a new anomaly has a higher score than any anomaly in the past, the anomaly 
+scores are adjusted on a range from 0 to 100 based on the new data. This process 
+is called renormalization. It can mean rewriting a large number of documents in 
+the results index. Renormalization happens for results from the last 30 days or 
+100 bucket spans (depending on which is the longer) by default. If you have a 
+lot of results, generate many anomalies, and have a short bucket span, consider 
+setting `renormalization_window_days` to a lower value. For example, for a large 
+job renormalization could involve rewriting half a million documents for the 
+last 30 days, so it would be reasonable to set `renormalization_window_days` to 
+14 days, so the workload is reduced.
+
+
+[[short-bucket-span]]
+== Avoiding short bucket span for large jobs
+
+If you have a large amount of data that requires a short bucket span, consider 
+splitting it up to avoid having high cardinality. The combination of short 
+bucket span and high cardinality is a resource-intensive situation that you may 
+want to avoid.
```

Comment:
Using a longer bucket span is a quicker win. e.g. From 1m to 5m. We should recommend this first, ahead of splitting the job up.